linux/fedora-rpm/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM fedora:latest
```

Comment:
Fedora rawhide may become broken unexpectidly, maybe better base on a specified release of Fedora

Reply:
`fedora:latest` is Fedora 34. Rawhide is `fedora:rawhide`